test-suite/src/test/scala/org/scalajs/testsuite/jsinterop/ExportsTest.scala

```diff
@@ -1372,3 +1378,18 @@ object ExportObjSetterNamed_= { // scalastyle:ignore
   @JSExport
   val x = 1
 }
+
+@JSExport
```

Comment:
I put it there originally to be able to intercept the the tree of the methods looked like. To figure out the exact shape of the broken accessors. Now I should be able to inline the `def`s in the `it`. I believe the the bug will still appear.